src/System.Private.CoreLib/shared/System/Globalization/CompareInfo.Windows.cs

```diff
@@ -5,12 +5,38 @@
 using System.Buffers;
 using System.Diagnostics;
 using System.Runtime.InteropServices;
+using System.Runtime.CompilerServices;
 
 namespace System.Globalization
 {
     public partial class CompareInfo
     {
-        private unsafe void InitSort(CultureInfo culture)
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
This is not a performance critical method. It should not be marked as AggressiveInlining

Reply:
ok, I'll remove it.